src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/DiagnosticsOverlay.xaml

```diff
@@ -355,14 +354,46 @@
 					
 					<!--Label & Value -->
 					<Run Text="UC:" />
-					<Run Text="{Binding UncollectedCommands}" />
+					<Run Text="{Binding Counters.UncollectedCommands}" />
 					
 					<!-- Tool Tip -->
 					<ToolTipService.ToolTip>
 						<ToolTip Style="{StaticResource DiagnosticsCounterToolTipStyle}"
 								 Content="Number of uncollected commands (Disposed - Destroyed)" />
 					</ToolTipService.ToolTip>
 				</TextBlock>
+
+				<!-- Private Memory Size -->
+				<!-- Not available on iOS for now. -->
+				<!-- See https://github.com/dotnet/runtime/issues/86251 for more details. -->
+				<not_ios:TextBlock Grid.Row="1"
+								   Foreground="OrangeRed"
```

Comment:
I would avoid going into "danger" colors and stay within something that doesn't suggest a warning or error. Probably just different shades of blue would be better.

Reply:
I'd go for lighter blues since the current one is already darkish.